custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/util/MavenHelper.java

```diff
@@ -63,6 +67,17 @@ public int run(File buildDir, boolean failOnError, String ... args) throws IOExc
         return runFor(buildDir, callArgs.toArray(args));
     }
 
+    private String getOsSpecificMavenCommand() {
+        String mvnCmd = "mvn";
+
+        String osName = System.getProperty("os.name");
+        if(osName.toLowerCase().compareTo("windows") == 0) {
```

Comment:
In edge cases it may be null. 

Maybe this one?

```
if(osName != null && os.toLowerCase().contains("windows")) { 
```

Reply:
fixed